src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -277,24 +277,9 @@ private SocketAsyncEngine()
                     throw new InternalException(err);
                 }
 
-                //
-                // Start the event loop on its own thread.
-                //
-                bool suppressFlow = !ExecutionContext.IsFlowSuppressed();
-                try
-                {
-                    if (suppressFlow) ExecutionContext.SuppressFlow();
```

Comment:
Why did you remove the execution context stuff? That applies to Thread equally.

Reply:
I assumed otherwise. Adding this back.